Project/Controllers/InOutController.cs

```diff
@@ -1,19 +1,29 @@
-using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
 using Project.Models;
+using System.Diagnostics;
 
-namespace Project.Controllers;
+namespace JsonParsing;
 
 public class InOutController : Controller
 {
+    [HttpPost]
+    public IActionResult AddMessage(string InputMessage)
+    {
+        FileStream file = System.IO.File.OpenRead("src/data.json");
+        var InOutModel = new InOutModel(file);
+        InOutModel.AddMessage("Guest", InputMessage, "src/data.json");
```

Comment:
Actions related to files could be added to helper class. Would make the code more readable.